heron/stmgr/src/cpp/manager/stmgr-client.cpp

```diff
@@ -199,7 +199,8 @@ void StMgrClient::SendStartBackPressureMessage() {
   REQID_Generator generator;
   REQID rand = generator.generate();
   // generator.generate(rand);
-  proto::stmgr::StartBackPressureMessage* message = acquire(message);
+  proto::stmgr::StartBackPressureMessage* message = acquire(
+    static_cast<proto::stmgr::StartBackPressureMessage *>(nullptr));
```

Comment:
People should read the definition of acquire() and then figure out the type deduction. So I don't think it is a problem.